extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```diff
@@ -810,6 +810,36 @@ describes.realWin('amp-ad-network-doubleclick-impl', realWinConfig, env => {
       impl.getAdUrl(CONSENT_POLICY_STATE.UNKNOWN_NOT_REQUIRED).then(url => {
         expect(url).to.not.match(/(\?|&)npa=(&|$)/);
       }));
+
+    it('should include msz/psz if in experiment', () => {
+      sandbox.stub(impl, 'randomlySelectUnsetExperiments_').returns(
+          {flexAdSlots: '21063174'});
+      impl.setPageLevelExperiments();
+      return impl.getAdUrl().then(url => {
+        expect(url).to.match(/(\?|&)msz=[0-9]+x-1(&|$)/);
+        expect(url).to.match(/(\?|&)psz=[0-9]+x-1(&|$)/);
+        expect(url).to.match(/(=|%2C)21063174(%2C|&|$)/);
```

Comment:
Is there really no utility function for pulling out a url param?

[optional] would you be up for writing one?

Reply:
Not sure that would save us much, since we'd still have to verify the pulled out param. I can look into it in a follow up, though.